src/main/java/teammates/common/datatransfer/questions/FeedbackConstantSumQuestionDetails.java

```diff
@@ -65,49 +65,34 @@ public boolean extractQuestionDetails(
             Map<String, String[]> requestParameters,
             FeedbackQuestionType questionType) {
 
-        String distributeToRecipientsString = null;
-        String pointsPerOptionString = null;
-        String pointsString = null;
-        String pointsForEachOptionString = null;
-        String pointsForEachRecipientString = null;
-        String forceUnevenDistributionString = null;
-        boolean distributeToRecipients = false;
-        boolean pointsPerOption = false;
-        boolean forceUnevenDistribution = false;
-        int points = 0;
-
-        distributeToRecipientsString =
```

Comment:
Why not just add `String` in front of `distributeToRecipientsString`. This will get rid of the indentation issue.
Currently, the indentation is acceptable but not nice.